cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -89,12 +89,13 @@ spec:
   # enable the crash collector for ceph daemon crash collection
   crashCollector:
     disable: false
+  # automate [data cleanup process](https://github.com/rook/rook/blob/master/Documentation/ceph-teardown.md#delete-the-data-on-hosts) in cluster destruction.
   cleanupPolicy:
-    # cleanup should only be added to the cluster when the cluster is about to be deleted.
-    # After any field of the cleanup policy is set, Rook will stop configuring the cluster as if the cluster is about
-    # to be destroyed in order to prevent these settings from being deployed unintentionally.
-    # To signify that automatic deletion is desired, use the value "yes-really-destroy-data". Only this and an empty
-    # string are valid values for this field.
+    # confirmation indicates whether user really want the automatic cleanup. Only empty string(default)
```

Comment:
```suggestion
    # confirmation indicates whether user really wants the automatic cleanup. Only empty string(default)
```

Reply:
This sentence disappeared after reflecting Travis's comment.